applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp

```diff
@@ -50,7 +50,7 @@ void CrBeamElementLinear3D2N::CalculateLocalSystem(
   this->CalculateLeftHandSide(rLeftHandSideMatrix, rCurrentProcessInfo);
 
   Vector nodal_deformation = ZeroVector(msElementSize);
-  this->GetValuesVector(nodal_deformation);
+  CrBeamElement3D2N::GetValuesVector(nodal_deformation); // modified by M.Fusseder

```

Comment:
Please remove this, this blocks derivation, idem for the further

Reply:
Well, I was pointing this because can be a problem in the future

Reply:
I share Vicente's concern about this. Having said this i don't have a solution.

I think this point is worth a detailed discussion